job-service/pom.xml

```diff
@@ -269,6 +274,12 @@
             <version>7.1.0</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter</artifactId>
+            <version>RELEASE</version>
```

Comment:
Why are you specifying `<version>RELEASE</version>` here?

![image](https://user-images.githubusercontent.com/20627297/111404689-1db5ce00-86c7-11eb-8fc5-6dff36e49938.png)

Is your IDE warning you about this?

Reply:
No, I am not getting any warning about it. I see there was a case raised some time ago [here](https://youtrack.jetbrains.com/issue/IDEA-181023)

Reply:
Surprising that the two free IDEs both warn about this and yet the commercial one doesn't.  Ok.